src/pkg/control/options.go

```diff
@@ -11,6 +11,7 @@ type Options struct {
 	FailFast           bool            `json:"failFast"`
 	RestorePermissions bool            `json:"restorePermissions"`
 	ToggleFeatures     Toggles         `json:"ToggleFeatures"`
+	RepoConnect        bool            `json:"-"`
```

Comment:
I think `options` is the wrong owner for this flag.  The struct is used to aggregate runtime configurations: settings that could cascade to many different places throughout processing.  What you're solving in this PR is a one-off flag that only involves the repository.Connect call.  Placing the flag into options will expose it to a lot more topology, and that might confuse contributors who are working with options downstream.

My recommendation is to add the flag to the Connect() parameters themselves.  That way it's clear that this change only affects that one function call, and isn't going to cause confusion elsewhere.